docs/reference/execute-methods.md

```diff
@@ -199,11 +199,65 @@ bundleId | string | yes | The bundle identifier of the application to be removed
 
 #### Returned Result
 
-Either `true` if the app was successfully uninstalled, otherwise `false`
+Either `true` if the application was successfully uninstalled, otherwise `false`
 
 ### mobile: launchApp
 
-Executes the given app on the device under test. If the app is already running then it would be activated. If the app is not installed or cannot be launched then an exception is thrown.
+Executes the given application on the device under test. If the application is already running then it would be activated.
+If the application is not installed or cannot be launched then an exception is thrown.
+
+It accepts `arguments` and `environment` to start an application with them.
+
+An example of the `arguments` usage is language and locale.
```

Comment:
An an usage example `arguments` allow you to enforce language and locale for the application to start with.